docs/services/data/_index.md

```diff
@@ -25,11 +25,11 @@ images: ["/services/icons/data-management.svg"]
 no_service: true
 ---
 
-The data management service is a robust solution for smart machine data handling, [data capture](/services/data/capture/), and [cloud synchronization](/services/data/cloud-sync/).
+The data management service is a robust solution for handling machine data, allowing you to capture data, sync it to the cloud, and query it.
 Using the data management service, you can collect data from different parts of a robot, IoT device, or any other machine.
-Once collected, you can configure which data to securely sync to the cloud, and view, sort, or manage it directly in the cloud without needing to manually gather data from each machine.
+Once collected, you can configure which data to securely sync to the cloud, and view, query, or manage it directly in the cloud without needing to manually gather data from each machine.
```

Comment:
```suggestion
Once collected, you can securely sync data to the cloud, and view, query, or manage it directly in the cloud without needing to manually gather data from each machine.
```
I'm going to say "securely" instead of "selectively" because I think "selectively" could be interpreted in some odd ways